prybar_assets/nodejs/input-sync.js

```diff
@@ -70,8 +70,40 @@ function ensureRawMode(cb) {
   return ret;
 }
 
-const delChar = "\x7f";
-const escapeClearLineRight = "\x1b[K";
+/**
+ * Handles ANSI escapes from stdin.
```

Comment:
since these constants are only used once, I just dropped them + added comments where they're used.

Reply:
Even if only used once, it's preferable to keep all magic variables in constants. `case leftArrowKey` is a lot clearer to a reader than `case "[D"`. I think it'd be best to bring back the constants (Plus add new ones for the new magic variables), while keeping the comments. That way the code is easier to parse and is documented.